2024/json-ld-wg/index.html

```diff
@@ -203,8 +203,30 @@ <h2>Scope</h2>
 
 
         <ol>
-          <li>[Placeholder: safe mode]</li>
-          <li>[Placeholder: context file hash, and related stuff]</li>
+          <li>
+            JSON-LD has become a common structural foundation for a wide range of data models. Some of 
+            those environments have more restrictive requirements on how unmapped terms should be treated (i.e., 
+            when JSON keys are present only in the document, but not in the context definition). The current 
+            JSON-LD API stipulates that the unknown term is silently ignored. The Working Group will specify 
+            a "safe mode", that will detect these situations and cause the processing to fail. This mode is 
+            already available in some JSON-LD API implementations, such as 
+            <a href="https://github.com/digitalbazaar/jsonld.js?tab=readme-ov-file#safe-mode">jsonld.js</a> and 
+            <a href="https://dotnetrdf.org/2025-07-03-dn4-3-4-0-released/#:~:text=This%20release%20also%20introduces%20a,rather%20than%20silently%20ignoring%20them.">dotNetRDF</a>, 
+            and is especially useful if the resulting RDF Graph is to be canonicalized.
+          </li>
+          <li>
+            External (i.e., non-inline) JSON-LD context files are identified using URIs. This is a common
+            pattern across many technologies that has, at times, confused implementers who 
+            expect that they should always use these identifiers as locators, and dereference them. This
+            assumption can create risks and vulnerabilities. 
```

Comment:
I suggest we add:
```suggestion
            assumption can create risks and vulnerabilities, that have been documented previous versions of JSON-LD, but also often overlooked.
```
Many people criticize JSON-LD on the basis of those risks, ignoring the fact that they were already described in JSON-LD 1.0, with mitigation strategies...

Reply:
fix to @pchampin's suggestion
```suggestion
            assumption can create risks and vulnerabilities that have been documented, but also often overlooked, in previous versions of JSON-LD.
```